### ISSUE TITLE
SkillAPI[Free] triggers MCMMO passive skills?

### ISSUE THREAD

**Issue by Yaemus** [Eniripsa96/SkillAPI#230]
There's something I've been trying to get straight: I've been trying to figure out a way to disable or find the reason why McMMO skills are being triggered by SkillAPI ( The free version ) skills that I've made even by skills that don't deal any damage(Albeit the sword passive skills are the main culprit here for inflicting bleed constantly.), Part of me thinks it has to deal with the damage type SkillAPI passes through, but I am completely uncertain of it as another part of me wonders if the Premium version's "Damage classifications" can be used to prevent that from happening? I don't know at all and this is pretty much speculation that needs clarification.

To summarize: I don't want the SkillAPI skills, passives or non-offensive skills that don't do any damage to trigger MCMMO's skills. Is there any way I can accomplish this? Or will this be impossible to do & I'm just not realizing a plugin is having an interaction with this? I'm honestly surprised this has never been well mentioned, if at all before.

**Comment by AngDrew** [comment]
I think that would be useless
Why you trigger mcmmo with other plugin if you can trigger it easier with mcmmo
What do you want to do?
Add a hotkey or something?
Mcmmo is passive, and some equipment have active skills on right click
But once more, that would be easier if you didnt trigger mcmmo by other plugin except you have something..

The answer is impossible but you can wait for eniripsa to answer if this will be added in future

**Comment by Yaemus** [comment]
I think you're missing the point. Lets say I cast a stream of firebolts with a sword out, Lets say my Swords skill is 1000, 75% of those firebolts will inflict bleed for no reason whatsoever. I want to know if SkillAPI is using a similar damage trigger to what McMMO uses.

**Comment by AngDrew** [comment]
You can add repeat for bleed

So just use one of them skillapi or mcmmo :D
And i recomend for you to use skillapi because you can customize a lot of things with the robust editor

**Comment by Yaemus** [comment]
I am aware of that as I've done Damage over time effects before, but it just won't be the same to some way as I have a love for both plugins, and disabling said skills would take quite bit of time to edit by having them level without using SP.

 Thanks for suggesting some alternatives though, I'll just have to see if the creator can confirm what I'm experiencing.

**Comment by Devillion** [comment]
@Yaemus I have a discord community for this plugin and other rpg like plugins, we have 60+ members and most are active.

come ask your questions on our discord: https://discord.gg/UwJMXGv

**Comment by Eniripsa96** [comment]
If McMMO is listening to any damage a player does to trigger their passives, what you're describing would happen and I wouldn't be able to change that. What you could try doing is requesting McMMO to ignore damage events when SkillAPI abilities are dealing damage. They would just need the following check to ignore skill damage:

`if (Skill.isSkillDamage()) return;`

Will close out this issue since I can't do anything about it myself.
